lib/GoogleAuthenticator.php

```diff
@@ -93,13 +93,14 @@ public function getCode($secret, $time = null)
      * @param string $user
      * @param string $hostname
      * @param string $secret
+     * @param string $issuer
      *
      * @return string
      */
-    public function getUrl($user, $hostname, $secret)
+    public function getUrl($user, $hostname, $secret, $issuer = '')
```

Comment:
This is a BC break for extending classes. Please use func_get_args() to retrieve the new argument, and add a NEXT_MAJOR comment explaining what to do on next major release